yarn.lock

```diff
@@ -16255,7 +16255,7 @@ pouchdb-adapter-leveldb-core@7.0.0:
     sublevel-pouchdb "7.0.0"
     through2 "2.0.3"
 
-pouchdb-adapter-memory@7.0.0:
+pouchdb-adapter-memory@7.0.0, pouchdb-adapter-memory@^7.0.0:
```

Comment:
I'm not sure what's going on here but it looks wrong to me.

Reply:
good eye again.

Reply:
thanks!